W-16993325-pce4-documentation-kt (#344)
mulesoft/docs-private-cloud: modules/ROOT/pages/backup-and-disaster-recovery.adoc

To preserve your data:

* Enable automatic backups to protect against system failures or operational issues.
* Schedule backups at least daily, or more frequently based on your needs.
* Store backups externally, outside the Anypoint Platform cluster.

The backup and restore process is controlled via Anypoint Platform PCE APIs.

== Create a Backup

To create a backup, run the following command:

`curl -k <platform-dns>/platform/backup-X POST -H 'Authorization: Bearer ${token}' -H 'Content-Type: application/json' -d '{"backup-file-name": "${backupFileName}.tar.gz", "nfs-server": "${fileSystemDns}", "nfs-path": "/"}'`

=== Backup Content

[%header%autowidth.spread]

|===
| Information | Included in Anypoint Platform PCE Backup 
| Databases | Yes
| Configurations | Yes
| Secrets | Yes
| Object Store | Yes
| Disclaimer settings | Yes
| SMTP settings | Yes
| DNS settings | No
| Platform certificate | No
| Ops Center certificate | No
| NFS settings | No
| NFS files | No	
| Drives | No
| Runtimes* | No	
| Anypoint Monitoring metrics | No
|===
 
*Anypoint Platform PCE has information only about runtimes, such as their names and status.

== Perform a System Restore

> Consider the following points before you perform a system restore:

Reviewer: ```suggestion
Consider the following before you perform a system restore:
```

> * Before running a restore, you must manually configure certain settings, such as NFS, certificates, and DNS, as they are not included in the backup.

Reviewer: ```suggestion
* Before running a restore, configure settings such as NFS, certificates, and DNS because they are not included in the backup.
```
